src/components/views/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,33 @@
+import {HomeView, MainPage} from '../index';
+import {NavigationContainer} from '@react-navigation/native';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+import {ROUTES} from '../../../static/types/routeTypes';
+import auth from '@react-native-firebase/auth';
+
+const Stack = createNativeStackNavigator();
+
+export const Navigation = () => {
+  return (
+    <NavigationContainer>
+      <Stack.Navigator
```

Comment:
Ta nawigacja nie jest zbyt dobrze zrobiona ogólnie, powinieneś mieć tab'y, a w taback stack'i i wszystko mieć zadeklarowane tutaj. 

Każdy TAB to jest osobny element "menu", który ma własna historię, taby się nie przykrywają, np.
Home screen, settings, search -> to są TABY

Każdy tab ma stack, jak nazwa mówi, stacki się "stackują na sobie" (przy np. cofaniu jak swipe robisz na iOS albo na androidzie jak to włączysz, to widać, że jest screen pod tym co z niego cofasz, to jest stack)


Np.

Masz ekran HOME, na ekranie home masz elementy, do których detali możesz przejść (/details), klikasz w jedne, klikasz w drugie, ten details screen to jest STACK, i wtedy masz historię -> Home -> details/1 -> details/2

każdy tab ma osobną historie

Stack może być montowany kilka razy w różnych tabach jak coś


No i teraz tak, masz to swoją rejestracje, główny ekran powinien być tab'em, każdy step rejestracji, powinien być stack'iem